spec/status.yml

```diff
@@ -206,6 +206,46 @@ components:
         security:
           - conjurAuth: []
 
+    JWTAuthenticatorStatus:
```

Comment:
Looks like this endpoint might already be covered by the existing `ServiceAuthenticatorStatus` endpoint definition.

Reply:
I understand your point, but dont we need to specify anyway to represent the separate JWT status route in conjur repository?

Reply:
The route isn't separate though, it is already fully described by the existing `ServiceAuthenticatorStatus` endpoint. We have separate endpoints for each authenticator in the `authentication.yml` file because they cannot all be accurately described with a single one.